### DISCUSSION TITLE
SQL functions development plan

### DISCUSSION THREAD

**Discussion by ZeaLoVe** [databendlabs/databend#7309] [General]
# Summary

The functions already support can be found in https://databend.rs/doc/reference/functions/.
But I think better to have a place showing the development plan, if a function is important but lack in current version, we need to catch up, if a function can implement with other ways we had, better to have a doc about it.

# Important functions (not implement yet)

# Will Not Implement(have a way to replace it)

cc @sundy-li

**Comment by sundy-li** [comment]
Yes, but every DBMS had its own function list. Which list do you want to refer to?

**Reply by ZeaLoVe** [reply]
Maybe we can collect and paste it here for further discussion? This is more a product issue than a technical one.

**Comment by ZeaLoVe** [comment]
MySQL common functions we basically support: https://dev.mysql.com/doc/refman/8.0/en/string-functions.html
